packages/dd-trace/test/proxy.spec.js

```diff
@@ -406,5 +457,35 @@ describe('TracerProxy', () => {
         expect(returnValue).to.equal(proxy)
       })
     })
+
+    describe('appsec', () => {
+      describe('trackUserLoginSuccessEvent', () => {
+        it('should call the underlying NoopAppsecSdk method', () => {
+          const user = { id: 'user_id' }
+          const metadata = { metakey1: 'metavalue1' }
+          proxy.appsec.trackUserLoginSuccessEvent(user, metadata)
+          expect(appsecSdk.trackUserLoginSuccessEvent).to.have.been.calledOnceWith(user, metadata)
```

Comment:
I usually use `.calledOnceWithExactly()` just to be sure I'm not forgetting something

Reply:
yes, it is more explicit 👍